### ISSUE TITLE
Should hand values be read out if one is at the top?

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/ratio-and-proportion#317]
**Test device**
Dell
**Operating System**
Win 10
**Browser**
Chrome
**Problem description**
For https://github.com/phetsims/QA/issues/582. Found with JAWS.
When tick marks are on and the screen reader is reading numbers, it will read out the numbers of both hands as you move them. However, if one is at the top, it does not seem to do that. I noticed this while the hands were locked in ratio. When the right hand hit the top, it said the right hand was at the top, but did not say the value of that hand, nor what the left hand value was.

**Visuals**
https://drive.google.com/file/d/183Q3QoW5dQRmAdLoZM2cxwjYEMETcQLX/view?usp=sharing

Troubleshooting information:

<details>

!!!!! DO NOT EDIT !!!!!
Name: ‪Ratio and Proportion‬
URL: https://phet-dev.colorado.edu/html/ratio-and-proportion/1.0.0-dev.77/phet/ratio-and-proportion_all_phet.html
Version: 1.0.0-dev.77 2020-12-08 00:32:10 UTC
Features missing: applicationcache, applicationcache, touch
Flags: pixelRatioScaling
User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/87.0.4280.88 Safari/537.36
Language: en-US
Window: 1280x658
Pixel Ratio: 1.5/1
WebGL: WebGL 1.0 (OpenGL ES 2.0 Chromium)
GLSL: WebGL GLSL ES 1.0 (OpenGL ES GLSL ES 1.0 Chromium)
Vendor: WebKit (WebKit WebGL)
Vertex: attribs: 16 varying: 30 uniform: 4096
Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 32)
Max viewport: 32767x32767
OES_texture_float: true
Dependencies JSON: {}

</details>

**Comment by terracoda** [comment]
This issue is a duplicate issue. We plan to make this edge response to fire not on the last step, but on a step after the last step in #247

**Comment by terracoda** [comment]
I am going to go ahead and close this issue because it will be worked on in #247.
